source/gui/installerGui.py

```diff
@@ -404,34 +404,46 @@ def __init__(self, parent):
 
 	def onCreatePortable(self, evt):
 		if not self.portableDirectoryEdit.Value:
-			# Translators: The message displayed when the user has not specified a destination directory
-			# in the Create Portable NVDA dialog.
-			gui.messageBox(_("Please specify a directory in which to create the portable copy."),
+			gui.messageBox(
+				# Translators: The message displayed when the user has not specified a destination directory
+				# in the Create Portable NVDA dialog.
+				_("Please specify a directory in which to create the portable copy."),
+				# Translators: the title of an error dialog.
 				_("Error"),
-				wx.OK | wx.ICON_ERROR)
+				wx.OK | wx.ICON_ERROR
+			)
 			return
-		if not os.path.isabs(self.portableDirectoryEdit.Value):
+		expandedPortableDirectory = os.path.expandvars(self.portableDirectoryEdit.Value)
+		if not os.path.isabs(expandedPortableDirectory):
 			gui.messageBox(
-				# Translators: The message displayed when the user has not specified an absolute destination directory
-				# in the Create Portable NVDA dialog.
-				_("Please specify an absolute path (including drive letter)  in which to create the portable copy."),
-				# Translators: The message title displayed
-				# when the user has not specified an absolute destination directory
-				# in the Create Portable NVDA dialog.
+				_(
+					# Translators: The message displayed when the user has not specified an absolute destination directory
+					# in the Create Portable NVDA dialog.
+					"Please specify an absolute path (including drive letter)  in which to create the "
+					"portable copy. It may include system variables (%temp%, %userprofile%, etc.)."

```

Comment:
I think this could be simplified.
I think if someone is aware of the meaning of "absolute path" they are aware that it includes the drive letter.
It also seems like the logic below doesn't require a drive letter.

```suggestion
					"Please specify the absolute path where the portable copy should be created. "
					"It may include system variables (%temp%, %userprofile%, etc.)."
```